hubble-fe/src/index.less

```diff
@@ -70,8 +67,7 @@ body {
       url(./assets/imgs/ic_biaoge_hover.svg)
       url(./assets/imgs/ic_biaoge_pressed.svg)
       url(./assets/imgs/ic_json_normal.svg) url(./assets/imgs/ic_json_hover.svg)
-      url(./assets/imgs/ic_json_pressed.svg)
-      url(./assets/imgs/ic_close_16.svg)
+      url(./assets/imgs/ic_json_pressed.svg) url(./assets/imgs/ic_close_16.svg)
```

Comment:
need to wrap line before the second "url"

Reply:
ok